Mathlib/Data/List/Basic.lean

```diff
@@ -1234,6 +1233,9 @@ theorem get?_length (l : List α) : l.get? l.length = none := get?_len_le le_rfl
 theorem nthLe_mem (l : List α) (n h) : nthLe l n h ∈ l := get_mem ..
 #align list.nth_le_mem List.nthLe_mem
 
+theorem nthLe_congr {l : List α} {n p : ℕ} {h : n < length l} (hnp : n = p) :
+    nthLe l n h = nthLe l p (hnp ▸ h) := by simp [hnp]
+
```

Comment:
What's the deal here?

Reply:
This is the sole appearance of this theorem, so I suspect its presence here is a merge mistake?

Reply:
I'll remove this assuming things build locally.